docs/guided-tour.md

```diff
@@ -56,29 +56,36 @@ During this tutorial, you're going to try out some cool features of Sandstorm. Y
 
 One of the common reasons to use Sandstorm is to have online collaboration software that runs on your own server.
 
-Etherpad is a real-time text editor that demonstrates how to use Sandstorm that way. Make sure you're logged into Sandstorm (if using the demo, click **Start the demo**). Then click:
+Etherpad is a real-time text editor that demonstrates how to use Sandstorm that way. Make sure you're logged into Sandstorm (if using the demo, click **Try a quick demo**). Then:
 
-* **Install apps** - this takes you to the Sandstorm app list. (Soon to be the Sandstorm App Marketplace!)
-* Click the **Install** button next to _Etherpad_. (Sandstorm may ask you for confirmation.)
-* Click **New Etherpad Document** button on your Sandstorm.
+* Click **Install...** - this takes you to the Sandstorm App Market.
+* Find _Etherpad_ and click the **Install** button. Sandstorm will ask you for confirmation.
+* Click **Install Etherpad**.
+* Once Etherpad is installed, click **Create new pad**.
 
 You're now in an Etherpad document. Installing Etherpad allows you to create as many Etherpad documents as you want.
 
-If you want to create spreadsheets on your server, consider installing _Ethercalc_.
+If you want to create spreadsheets on your server, consider installing [Ethercalc](https://apps.sandstorm.io/app/a0n6hwm32zjsrzes8gnjg734dh6jwt7x83xdgytspe761pe2asw0).
 
-If you want to edit scientific documents, consider installing _ShareLaTeX_.
+If you want to edit scientific documents, consider installing [_ShareLaTeX_](https://apps.sandstorm.io/app/5vuv7v0w7gu20z72m78n83rx9qqtqpmtk32f39823wh967z226qh).
```

Comment:
To be consistent, if you're going to not italicize Ethercalc above, then also don't italicize ShareLaTeX.